app/src/main/AndroidManifest.xml

```diff
@@ -9,21 +9,26 @@
     <application
             android:name=".MoviesApp"
             android:allowBackup="false"
-            android:icon="@mipmap/ic_launcher"
+            android:icon="@drawable/baseline_movie_white"
             android:label="@string/app_name"
             android:networkSecurityConfig="@xml/network_security_config"
-            android:roundIcon="@mipmap/ic_launcher_round"
+            android:roundIcon="@drawable/baseline_movie_white"
             android:supportsRtl="true"
             android:theme="@style/AppTheme"
             tools:targetApi="n">
-        <activity android:name="com.mho.training.features.main.MainActivity">
+        <activity
+                android:name="com.mho.training.features.splash.SplashActivity"
+                android:theme="@style/SplashTheme">
             <intent-filter>
-                <action android:name="android.intent.action.VIEW" />
                 <action android:name="android.intent.action.MAIN" />
-
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name="com.mho.training.features.main.MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
```

Comment:
What this implementation do?

Reply:
SplashActivity is now the launcher activity, so MainActivity keeps the action view. Also, the compilator throws:

_App is not indexable by Google Search;  At least one Activity with an ACTION-VIEW intent-filler._